src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -59,22 +59,17 @@ const ProgressiveProfiling = (props) => {
     ensureAuthenticatedUser(DASHBOARD_URL)
       .then(() => {
         hydrateAuthenticatedUser().then(() => {
+          identifyAuthenticatedUser(authenticatedUser?.userId);
```

Comment:
@shahbaz-arbisoft could you please tell why are we adding it here instead of the useEffect that we were using?